core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -240,7 +242,7 @@ class CivilizationInfo {
         // This is the only thing that is NOT switched out, which makes it a source of ConcurrentModification errors.
         // Cloning it by-pointer is a horrific move, since the serialization would go over it ANYWAY and still lead to concurrency problems.
         // Cloning it by iterating on the tilemap values may seem ridiculous, but it's a perfectly thread-safe way to go about it, unlike the other solutions.
-        toReturn.exploredTiles.addAll(gameInfo.tileMap.values.asSequence().map { it.position }.filter { it in exploredTiles })
+        toReturn.exploredTiles = exploredTiles
```

Comment:
Wouldn't this change invalidate the comments above?

Reply:
Correct